pkg/ingress/apisix/upstream_test.go

```diff
@@ -94,6 +97,7 @@ kind: ApisixUpstream
 apiVersion: apisix.apache.org/v1
 metadata:
   name: httpserver
+  namespace: cloud
```

Comment:
Still use cloud namespace? we should use ingress-apisix.

Reply:
This is a test-case

Reply:
OK